### PR TITLE
Add talk sidebar tab for files

### PR DESCRIPTION
**Pull request by ma12-co**
Screenshot-20191212152349-855x497

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/TalkSidebarTab.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import TalkSidebarTab from './TalkSidebarTab'
+window.addEventListener('DOMContentLoaded', () => {
+	if (OCA.Files && OCA.Files.Sidebar) {
+		OCA.Files.Sidebar.registerTab(new OCA.Files.Sidebar.Tab('Chat', TalkSidebarTab))
```
Comment (skjnldsv) [inline src/TalkSidebarTab.js]: maybe a more casual key like `talk-chat` :)
This is only used as unique key, not displayed or anything

Comment (ma12-co) [reply src/TalkSidebarTab.js]: allrighty :)

File: webpack.common.js [hunk 2]
```diff
@@ -11,7 +11,8 @@ module.exports = {
 		'admin/stun-server': path.join(__dirname, 'src', 'StunServerSettings.js'),
 		'admin/turn-server': path.join(__dirname, 'src', 'TurnServerSettings.js'),
 		'collections': path.join(__dirname, 'src', 'collections.js'),
-		'talk': path.join(__dirname, 'src', 'main.js')
+		'talk': path.join(__dirname, 'src', 'main.js'),
+		'appSidebarTab': path.join(__dirname, 'src', 'TalkSidebarTab.js'),
```
Comment (skjnldsv) [inline webpack.common.js]: kebab case for files as it's usually more readable :woman_shrugging: 
```suggestion
		'app-sidebar-tab': path.join(__dirname, 'src', 'TalkSidebarTab.js'),
```

Comment (ma12-co) [reply webpack.common.js]: Vue files are camelCased, do you refer to .js files only here?

Comment (skjnldsv) [reply webpack.common.js]: yes :)

Comment (skjnldsv) [reply webpack.common.js]: just the compiled name

**Pull request closed without merging**